### DISCUSSION TITLE
加油，这是一个很好的项目

### DISCUSSION THREAD

**Discussion by gitlsl** [UlyssesWu/XamarinPosed#2] [General]
市面上都没有c#版本的xposed模块，这个很有意思

**Comment by UlyssesWu** [comment]
谢谢支持~如遇到什么问题欢迎反馈。
（因为Xamarin的内部API在变动，如要测试请使用最新的VS，目前是VS 16.9）

目前我把它用在手机wx hook上，功能上完全满足需求，
只是不知是VXP的原因、还是异常处理的原因、亦或是手机内存太小，总之（长时间后台待机状态的）稳定性还是有待提升。

**Comment by gitlsl** [comment]
我也是想搞wx hook
注入后 启动  web server， 远程调用各种发消息  发朋友圈啥的， 可以搞吗？ 你是hook了做什么

要用的话是不是直接 引用XPosedAPI编译出来的dll    再引用那个jar 就可以了

**Comment by UlyssesWu** [comment]
实现功能需要修改XamarinPosed这个项目里的Loader.cs，里面有Xposed的接口实现。
http://117.51.146.36/post/xamarin-posed

我只是做一些收发消息，自动回复的功能，没做别的。
手机上启动web server可是有点费，我这边是让手机app启动时去连接服务器，由服务器控制收发，这样服务器上也可以通过插件增加处理功能。
